models/Subscriber.js

```diff
@@ -36,25 +36,17 @@ SubscriberSchema.statics.sendMessage = function(message, url, callback) {
             var options = {
                 to: subscriber.phone,
                 from: config.twilioNumber,
-                body: message
+                body: message,
+                messagingServiceSid: 'MG9752274e9e519418a7406176694466fa',
```

Comment:
What is the purpose of this messageServiceSid? Seems like this one doesn't belong here.